project_task_analytic_tag/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Project Task Analytic Tag
+=========================
+
+This module extends the functionality of project and mrp_repair to support to
```

Comment:
mrp_repair?
It is not in dependency list and anyway would simply need another module if you'd wish to do so.